### DISCUSSION TITLE
corfu-index-mode selection position issue

### DISCUSSION THREAD

**Discussion by pinacle2000** [minad/corfu#511] [Q&A]
I use doom emacs with module  (corfu +orderless +dabbrev +icons). It works fine until I tried corfu-index-mode. As can be seen in the figure, I tried to get Number-3 "styles" with C-3 tab, but got the Number-2 item. I also tried C-0, which gave nothing.

![image](https://github.com/user-attachments/assets/fd334b2d-0cd9-42d4-911f-b41a7690d915)

I checked doom module configuration, and found no corfu-index-mode related code, so I don't know whether it is an issue of corfu? Or can I configure the numbers starting from 1?

**Comment by minad** [comment]
I seem unable to reproduce this. Please try to reproduce without Doom. Thanks!

**Reply by pinacle2000** [reply]
I am novice to emacs so it's hard for me to use emacs without doom. I tried emacs -q, and package-install corfu, but can not make corfu work.

I also tried to disable the orderless and vertico modules in doom possibly related with the issue, but it remains the same behavior. I'll try to post this issue to doom, and see whether it could be reproduced there.
